core/src/edu/asu/c3simulator/widgets/Location.java

```diff
@@ -0,0 +1,50 @@
+package edu.asu.c3simulator.widgets;
+
+/**
+ * This class holds the x and y coordinates for any actor as a single object of type
+ * Location
+ * 
+ * @author Alyahya, Mohammed
+ */
+public class Location
```

Comment:
It looks like you and I both had a need for a utility like this. PrecisePoint is currently in master, but it is modeled after awt.Point, and is intended to have a very short lifecycle, or be used privately.

This looks like it would be safer for point objects with longer lifecycles.

We'll need to do some refactoring later to decide when to use which, and if we should combine the two classes.